cloud-aws/src/main/resources/definitions/datahub-minimal-polcy.json

```diff
@@ -0,0 +1,104 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateTags",
+        "ec2:DeleteTags",
+        "ec2:DescribeInstances",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeRegions",
+        "ec2:DescribeInstanceTypeOfferings",
+        "ec2:DescribeInstanceTypes",
+        "ec2:DescribeSubnets",
+        "ec2:DescribeKeyPairs",
+        "ec2:DescribeVpcs",
+        "ec2:DescribeImages",
+        "ec2:CreateSecurityGroup",
+        "ec2:DescribeSecurityGroups",
+        "ec2:AllocateAddress",
+        "ec2:AuthorizeSecurityGroupIngress",
+        "ec2:DeleteSecurityGroup",
+        "ec2:CreateLaunchTemplate",
+        "ec2:DeleteLaunchTemplate",
+        "ec2:DescribeLaunchTemplates",
+        "ec2:DescribeAddresses",
+        "ec2:ReleaseAddress",
+        "ec2:RunInstances",
+        "ec2:CreateVolume",
+        "ec2:DescribeVolumes",
+        "ec2:AttachVolume",
+        "ec2:AssociateAddress",
+        "ec2:StopInstances",
+        "ec2:StartInstances",
+        "ec2:DescribeRouteTables"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetBucketLocation"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "cloudformation:DescribeStacks",
+        "cloudformation:CreateStack",
+        "cloudformation:DeleteStack",
+        "cloudformation:DescribeStackEvents",
+        "cloudformation:DescribeStackResources",
+        "cloudformation:DescribeStackResource",
+        "cloudformation:DescribeStacks"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "cloudwatch:DescribeAlarms"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:PassRole",
+        "iam:ListInstanceProfiles"
+      ],
```

Comment:
Also needs the following one:
```
    {
      "Sid": "IdentityAccessManagementLimited",
      "Action": [
        "iam:CreateServiceLinkedRole"
      ],
      "Effect": "Allow",
      "Resource": [
        "arn:aws:iam::*:role/aws-service-role/*"
      ]
    },
```
Rationale: These special roles are needed for some services to act on behalf of the invoking user/role. For instance, `AWSServiceRoleForCloudWatchEvents` is needed to manipulate CW events (e.g. see https://jira.cloudera.com/browse/ENGESC-7128), and `AWSServiceRoleForAutoScaling` is needed to use EC2 Autoscaling, and perhaps `AWSServiceRoleForRDS` is needed for RDS usage. (Not sure about the latter, but it is also found in the `cdp-nonprod-datahub-policy.json` attachment of the ENGESC-7128 ticket.)

These roles can be preexisting in the target account, in which case we are happy as AWS will not have to recreate them, so there will be no complaints about `iam:CreateServiceLinkedRole`. On the other hand, trying to launch in a totally empty region or a brand new AWS account where these roles do not exist will manifest of lovely AWS errors. 😄